### PR TITLE
Document and test configuration help

### PR DESCRIPTION
**Pull request by timspainNERSC**
# Document and test configuration help
## Fixes \#349

---
# Change Description

The configuration help system needs to be documented, and a test to look at the outputs would also be useful.

---
# Test Description

A new test `testHelpConfig.sh`, a bash script that runs the `nextsim` binary several times to examine the output of the configuration help system.

---
# Documentation Impact

Added a document titled 'Getting Help' as `getting_help.rst` in the documentation subdirectory.

### CODE REVIEW

File: docs/getting_help.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+.. raw:: html
+
+Getting Help
+============
+
+You can get help on compiling and running NextSIM-DG in several ways:
+
+- The documentation that can be found alongside this document.
+- The built-in help system.
+- By emailing the NextSIM-DG team.
```
Comment (MarionBWeinzierl) [inline docs/getting_help.rst:12]: Provide email address here?

Comment (timspainNERSC) [reply docs/getting_help.rst:12]: @einola What email address would be best to use?

Comment (einola) [reply docs/getting_help.rst:12]: I can ask IT to set up nextsim@nersc.no or something similar. Emails sent to there can then be forwarded to me and/or @timspainNERSC. How does that sound?

Comment (timspainNERSC) [reply docs/getting_help.rst:12]: That sounds just fine.

Comment (timspainNERSC) [reply docs/getting_help.rst]: Email address added.

**Comment by einola** [comment]
`nextsim@nersc.no` set up and tested.

**Pull request merged**